README.rst

```diff
@@ -1 +0,0 @@
-Browse the branches to see the different notebooks I've been working on!
```

Comment:
Why do I seem to see this diff in multiple PRs? Is this intentional?

Reply:
Ah, that's because I don't copy the README over to the new branches